tests/installation/bootloader.pm

```diff
@@ -134,6 +135,11 @@ sub run() {
             # ZYPP... enables proxy caching
         }
 
+        if (my $remote = get_var("REMOTE_SLAVE")) {
+            type_string_slow " " . get_var("NETSETUP") if get_var("NETSETUP");
+            type_string_slow " nameserver=" . join(",", @{get_host_resolv_conf()->{nameserver}});
```

Comment:
please split this also into 2 lines. The perl style we're trying to use in tests is supposed to be readable to intermediate perl users